editProfilePage.php

```diff
@@ -0,0 +1,65 @@
+<?php  
+    include('config/init.php');
+    echologgedInHeader("edit profile");
+    $user = getUser($_SESSION['accountId']); 
+  //because defined here, don't have to use request
+if(isset($_REQUEST['newProfileEdit'])){
+    insertNewProfileEdit(
```

Comment:
In a perfect world you'd do some validation here to make sure they filled everything out, but that's not worth worrying about right now